packages/wdio-allure-reporter/src/index.js

```diff
@@ -110,15 +110,15 @@ class AllureReporter extends WDIOReporter {
         const currentTest = this.allure.getCurrentTest()
 
         if (!this.isMultiremote) {
-            const { browserName, deviceName, desired } = this.capabilities
-            let targetName = browserName || deviceName || cid
+            const { browserName, deviceName, desired, device } = this.capabilities
+            let targetName = device || browserName || deviceName || cid
```

Comment:
`device` should be after `browserName` and `deviceName` as it is a BS specific capability

```suggestion
            let targetName = browserName || deviceName || device || cid
```

Reply:
@christian-bromann the reason for keeping `device` at the start was since the UDID is returned in `deviceName` when run on BrowserStack.

In case of testing with Appium or any other cloud, the device property will be undefined which should select deviceName as per the logic.

Reply:
ok, makes sense then